### DISCUSSION TITLE
Problem with visit:device filter

### DISCUSSION THREAD

**Discussion by skymarocchinia** [plausible/analytics#1489] [Anything Else]
Hello
using the stats api and filtering by visit:device (Desktop, Tablet, Mobile) I always receive the same data for each device, even changing the dates.
This is an example of APi,
https://plausible.io/api/v1/stats/aggregate?site_id=xxx&period=30d&event:page==/house/609ac0e10fcc4a002f129c4d;visit:device==Mobile
Am I doing something wrong?
Thank you
Alessandro

**Comment by ukutaht** [comment]
Hi @skymarocchinia!

Are you self-hosting or using our hosted version?

**Reply by skymarocchinia** [reply]
Hello @ukutaht ,
first of all thank you for your help.
I'm using your hosted version
Alessandro

**Reply by ukutaht** [reply]
OK, please write to hello@plausible.io including the domain name of your site so we can debug the data issue. Thanks!

**Reply by skymarocchinia** [reply]
Ok, thank you
